db/schema.rb

```diff
@@ -233,8 +233,8 @@
   create_table "provider_users_providers", force: :cascade do |t|
     t.bigint "provider_id", null: false
     t.bigint "provider_user_id", null: false
-    t.datetime "created_at", default: -> { "now()" }, null: false
-    t.datetime "updated_at", default: -> { "now()" }, null: false
+    t.datetime "created_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "updated_at", default: -> { "CURRENT_TIMESTAMP" }, null: false
```

Comment:
Interesting. I have no reason to believe this won't work, but peculiar that it came up as part of this?

Reply:
this keeps flip-flopping as different people update the schema. I believe its a Postgres version thing. It's harmless but very irritating. I think I'm going to try upgrading Postgres. I'm on version 10 which is a bit behind some (though it's ahead of production).

Reply:
👍 Don't bother. I figured it must be something like that; it's not worth discussing or upheaving your setup, we can roll with it!

Reply:
rollback, checkout `schema.rb`, db:reset and re-migrate seems to 'fix it'